src/parser/validation.ts

```diff
@@ -69,11 +69,11 @@ const isTokenIdInteger = (row: CollectibleTransfer): string[] =>
   row.tokenId.isInteger() ? [] : [`Token IDs must be integer numbers: ${row.tokenId.toFixed()}`];
 
 const isTokenValueInteger = (row: CollectibleTransfer): string[] =>
-  !row.value || row.value.isNaN() || row.value.isInteger()
+  !row.amount || row.amount.isNaN() || row.amount.isInteger()
     ? []
-    : [`Value of ERC1155 must be an integer: ${row.value.toFixed()}`];
+    : [`Value of ERC1155 must be an integer: ${row.amount.toFixed()}`];
 
 const isTokenValueValid = (row: CollectibleTransfer): string[] =>
-  row.token_type === "erc721" || (typeof row.value !== "undefined" && row.value.isGreaterThan(0))
+  row.token_type === "erc721" || (typeof row.amount !== "undefined" && row.amount.isGreaterThan(0))
     ? []
-    : [`ERC1155 Tokens need a defined value > 0: ${row.value?.toFixed()}`];
+    : [`ERC1155 Tokens need a defined value > 0: ${row.amount?.toFixed()}`];
```

Comment:
What is the value of an erc1155 supposed to be?

Reply:
Here is my understanding of ERC1155 tokens:

ERC1155 is kind of a multi-token.
It can be a mix of fungible and non fungible tokens.
So its kind of multiple ERC20 and ERC721 contracts in one smart contract. Invented i.e. for blockchain games

For instance you could put all items of an game into ERC1155 tokens. There might be unique items such as clothing which would have the total amount of 1 but there may be coins of which 1 million exist all with the same id.

So I could call safeTransferFrom(safeAddress, receiverAddress, idOfCoin, 4000) to send 4000 coins to another address.